### ISSUE TITLE
[v5] Unable to login to ReportPortal after shell installation

### ISSUE THREAD

**Issue by Chris-Huszcza-CS** [reportportal/reportportal#804]
**Describe the bug**
I've installed reportportal following the instructions here: https://github.com/reportportal/shell-installation, however when trying to login to ReportPortal the UI just displays "An error occurred while connecting to server: Network Error"

**To Reproduce**
I've modified the 
[start_rp.sh.txt](https://github.com/reportportal/reportportal/files/4122402/start_rp.sh.txt) file so that `RP_SERVER_PORT=8080` instead of `RP_SERVER_PORT=3000`, as per https://github.com/reportportal/reportportal/issues/798

I was redirected to http://n.n.n.n:8080/#login, and then entered the details from here: https://github.com/reportportal/shell-installation#check-your-reportportal-installation-available

Worth noting that the screen did not display the list of services (UAT, Analyzer, etc)

**Expected behavior**
I expected to be able to log in and/or see the list of services.

**Screenshots**
![ReportPortalLoginError](https://user-images.githubusercontent.com/52416273/73265895-be001900-41cd-11ea-8057-1744f059a415.png)

Here are the logs I've genered:
[postgresql.log](https://github.com/reportportal/reportportal/files/4122425/postgresql.log)
[reportportal.log](https://github.com/reportportal/reportportal/files/4122427/reportportal.log)
[service-analyzer.log](https://github.com/reportportal/reportportal/files/4122428/service-analyzer.log)
[service-api.log](https://github.com/reportportal/reportportal/files/4122430/service-api.log)
[service-index.log](https://github.com/reportportal/reportportal/files/4122431/service-index.log)
[service-uat.log](https://github.com/reportportal/reportportal/files/4122433/service-uat.log)
[service-ui.log](https://github.com/reportportal/reportportal/files/4122434/service-ui.log)

**Desktop (please complete the following information):**
 - Host is RHEL 7.7
 - Client is Win 10, Firefox 72.0.2

**Comment by Yumfriez** [comment]
@Chris-Huszcza-CS As I see in the `reportportal.log`:
time="2020-01-28T12:35:14Z" level=fatal msg="Error preparing server: error opening listener: listen tcp :8080: bind: address already in use"

So 8080 port is already used by another service.
You can either free this port or change exposed port value in the `traefik.toml` to run on another port

**Comment by Chris-Huszcza-CS** [comment]
Hi @Yumfriez ,

I've gone back to the server (after a reboot), and checked for listening ports using `lsof -i -P -n | grep LISTEN` and there's nothing listening on port 8080 or 8090 initially.

Then I changed the traefik.toml so that entrypPoints.http address = 8090, ran `start_rp.sh` andre-ran `lsof -i -P -n | grep LISTEN`. I found that these results:
```
java      1430 elasticsearch  243u  IPv6  15159      0t0  TCP [::1]:9300 (LISTEN)
java      1430 elasticsearch  244u  IPv6  25625      0t0  TCP 127.0.0.1:9300 (LISTEN)
java      1430 elasticsearch  293u  IPv6  15166      0t0  TCP [::1]:9200 (LISTEN)
java      1430 elasticsearch  294u  IPv6  15167      0t0  TCP 127.0.0.1:9200 (LISTEN)
beam.smp  1432      rabbitmq    8u  IPv4  22311      0t0  TCP *:25672 (LISTEN)
beam.smp  1432      rabbitmq   48u  IPv6  23412      0t0  TCP *:5672 (LISTEN)
beam.smp  1432      rabbitmq   49u  IPv4  22335      0t0  TCP *:15672 (LISTEN)
sshd      1434          root    3u  IPv4  22097      0t0  TCP *:22 (LISTEN)
sshd      1434          root    4u  IPv6  22099      0t0  TCP *:22 (LISTEN)
postmaste 1472      postgres    3u  IPv6  18825      0t0  TCP [::1]:5432 (LISTEN)
postmaste 1472      postgres    4u  IPv4  14981      0t0  TCP 127.0.0.1:5432 (LISTEN)
epmd      1759      rabbitmq    3u  IPv4  10130      0t0  TCP *:4369 (LISTEN)
epmd      1759      rabbitmq    4u  IPv6  10131      0t0  TCP *:4369 (LISTEN)
master    1813          root   13u  IPv4  20666      0t0  TCP 127.0.0.1:25 (LISTEN)
master    1813          root   14u  IPv6  20667      0t0  TCP [::1]:25 (LISTEN)
traefik   2592          root    3u  IPv6  22411      0t0  TCP *:8090 (LISTEN)
traefik   2592          root    5u  IPv6  22412      0t0  TCP *:8081 (LISTEN)
service-i 2594          root    3u  IPv6  15185      0t0  TCP *:9000 (LISTEN)
java      2595          root   49u  IPv6  24638      0t0  TCP *:8585 (LISTEN)
java      2596          root   44u  IPv6  25722      0t0  TCP *:9999 (LISTEN)
service-u 2599          root    3u  IPv6  24622      0t0  TCP *:8080 (LISTEN)
```

I tried to navigate to http://host:8080/ui and was redirected to http://host:8080/ui/#notfound. And I tried to login again in the /login screen, but again was notified of a Network Error. Logs attached:
[reportportal.log](https://github.com/reportportal/reportportal/files/4126834/reportportal.log)
[service-analyzer.log](https://github.com/reportportal/reportportal/files/4126836/service-analyzer.log)
[service-api.log](https://github.com/reportportal/reportportal/files/4126837/service-api.log)
[service-index.log](https://github.com/reportportal/reportportal/files/4126838/service-index.log)
[service-uat.log](https://github.com/reportportal/reportportal/files/4126839/service-uat.log)
[service-ui.log](https://github.com/reportportal/reportportal/files/4126840/service-ui.log)
[start_rp.sh.txt](https://github.com/reportportal/reportportal/files/4126843/start_rp.sh.txt)
[traefik.toml.txt](https://github.com/reportportal/reportportal/files/4126844/traefik.toml.txt)

**Comment by Yumfriez** [comment]
@Chris-Huszcza-CS 
`UI` should be started at port 3000
Traefik - on any free port (8080 by default)
And even if `UI` is running on the port 3000, you still should go to the port where `Traefik` is deployed (because it's the main entry point for RP application)

**Comment by Chris-Huszcza-CS** [comment]
Hi @Yumfriez ,

Thanks for the information. It turns out I couldn't run Traefik on port 8080 because the service-analyzer is also starting up on that port (logs attached below), but when I switched UI back to port 3000 and change Traefik's HTTP port to 8082, then it all startup ok and I was able to log in.
[reportportal.log](https://github.com/reportportal/reportportal/files/4132882/reportportal.log)
[service-analyzer.log](https://github.com/reportportal/reportportal/files/4132883/service-analyzer.log)

Many thanks for the help, I'm happy for the issue to be closed, but not sure if you want to do anything on your side first (around the default ports).

Cheers,
Chris
